packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -778,6 +778,30 @@ let getSingleUseLegacyLogCallback = (logger: ITelemetryLoggerExt, type: string)
 	};
 };
 
+/**
+ * Load the stores from a snapshot and returns the runtime.
+ * @param params - An object housing the runtime properties:
+ * - context - Context of the container.
+ * - registryEntries - Mapping from data store types to their corresponding factories.
+ * - existing - Pass 'true' if loading from an existing snapshot.
+ * - runtimeOptions - Additional options to be passed to the runtime
+ * - containerScope - runtime services provided with context
+ * - provideEntryPoint - Promise that resolves to an object which will act as entryPoint for the Container.
+ * This object should provide all the functionality that the Container is expected to provide to the loader layer.
+ * @legacy
+ * @alpha
+ */
+export async function loadContainerRuntime(params: {
+	context: IContainerContext;
+	registryEntries: NamedFluidDataStoreRegistryEntries;
+	existing: boolean;
+	runtimeOptions?: IContainerRuntimeOptions;
+	containerScope?: FluidObject;
+	provideEntryPoint: (containerRuntime: IContainerRuntime) => Promise<FluidObject>;
```

Comment:
I see you're omitting requestHandler here, which while I'm certainly supportive of, I think we already know customers are still relying on this today right?  If the goal is to get folks off the class ASAP, I have to imagine it's going to get added back?

Reply:
good point, that might be overreach, really the things i want gone is the custom ctor.